code/ZAS/Airflow.dm

```diff
@@ -105,6 +105,8 @@ atom/movable/GotoAirflowDest(n)
 /mob/living/carbon/human/check_airflow_movable(n)
 	if(reagents.has_reagent(MEDCORES))
 		return FALSE
+	if(reagents.has_reagent(SOFTCORES))
```

Comment:
I should have caught this before -- here and in human_movement.dm, instead of this do 
`if(reagents.has_any_reagent(CORES)` at L106 (and L73) respectively

then in reagents.dm do this: `#define CORES list(SOFTCORES, MEDCORES)`

This will be more maintainable

Reply:
got it!